packages/react-components/src/templates/EventPage.tsx

```diff
@@ -42,6 +43,7 @@ const EventPage: React.FC<EventPageProps> = ({
   backHref,
   lastModifiedDate,
   calendar,
+  hideMeetingLink,
```

Comment:
Personally, I'd prefer to pass `JoinEvent: ReactNode` here - and then pass `null` if we don't want to render this.

The way this component is set up, it takes the props of all sub-components. If that is the case, I'd rather switch to passing the sub-components directly.

But I think that might be out of scope for this issue.